### ISSUE TITLE
Keyboard shortcuts for paramter hints not working

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52681]
Environment Details:
VSCode Version : 1.24.0

Additional Details:
MAS Violated: MAS2.1.1

Repro Steps:

Launch VS Code.
Open Keyboard Shortcuts.

Actual:
 Trigger Parameter Hints( Ctrl+Shift+Space and Ctrl+P) has different behaviors while both have same functionalities.

Expected:
Both the commands should have same behavior.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

Attachment for Reference: 
[A11y VSCode Keyboard Shortcuts .pptx](https://github.com/Microsoft/vscode/files/2127638/A11y.VSCode.Keyboard.Shortcuts.pptx)

MAS Reference
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Does this issue occur when all extensions are disabled?: Yes

**Comment by chrmarti** [comment]
![image](https://user-images.githubusercontent.com/9205389/41780162-434ef58e-7634-11e8-9087-c346a4c77cfa.png)

@sandy081 Is this a problem with the keybindings editor? The screencast shows the same commands several times.

**Comment by isidorn** [comment]
@AccessibilityTestingTeam-TCS I can not reproduce this with latest vscode insiders.
Did you customize your keybindgins? If yes, can you reproduce with latest vscode insiders?

**Comment by AccessibilityTestingTeam-TCS** [comment]
@isidorn checked on version 1.25.0, and still finding the same behavior.

**Comment by isidorn** [comment]
@AccessibilityTestingTeam-TCS please try vscode insiders as I have mentioned in my comment above https://code.visualstudio.com/insiders/
Thanks
